src/modules/site/site.service.ts

```diff
@@ -31,4 +34,23 @@ export class SiteService {
     const { URL: siteId, Sitestatus: status } = data.value[0].fields;
     return { siteId, status };
   }
+
+  async createSiteId(): Promise<string> {
+    const requestToCreateSiteId: MdmCreateNumbersRequest = this.buildRequestToCreateSiteId();
+    // TODO APIM-133: Do we want to wrap any errors in a new error?
+    const [{ maskedId: createdSiteId }] = await this.mdmService.createNumbers(requestToCreateSiteId);
```

Comment:
@avaitonis it might be best to answer this with knowledge of your POST /sites ticket. Errors from `mdmService.createNumbers` will be of type `MdmException` and will have the message `Failed to create numbers in MDM.`.

Do we want to re-wrap those errors into a message that says something more like `Failed to create a site ID` _here_? Or do we want to do anything like that in your ticket instead? (Personally, I think it'll make more sense in your ticket)

Reply:
Thanks, happy todo it my ticket